assets/js/base/components/radio-control-accordion/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { withInstanceId } from '@woocommerce/base-hocs/with-instance-id';
+
+/**
+ * Internal dependencies
+ */
+import RadioControlOption from '../radio-control/option';
+import '../radio-control/style.scss';
+
+const RadioControlAccordion = ( {
+	className,
+	instanceId,
+	id,
+	selected,
+	onChange,
+	options = [],
+} ) => {
+	const radioControlId = id || instanceId;
+
+	return (
+		options.length && (
+			<div
+				className={ classnames(
+					'wc-block-components-radio-control',
+					className
+				) }
+			>
+				{ options.map( ( option ) => {
+					const hasOptionContent = 'content' in option;
```

Comment:
usage of `in` here assumes `option` is an object. Should we defensively verify that first?

Reply:
It feels a bit excessive but for sure it would not hurt. Sometimes I feel like things like that should just break - because this implies the wrong usage of the component.

Reply:
Corrected.